tests/checks/anomalydetection.cc

```diff
@@ -396,8 +412,10 @@ TEST_F(AnomalydetectionCheck, StatusChanges) {
 
   previous = now;
   now = std::time(nullptr);
-  cmd = fmt::format("[{}] PROCESS_SERVICE_CHECK_RESULT;test_host;test_svc;0;service ok| "
-         "metric=71.7%;80;90;10;100", now);
+  cmd = fmt::format(
+      "[{}] PROCESS_SERVICE_CHECK_RESULT;test_host;test_svc;0;service ok| "
+      "metric=71,7%;80;90;10;100",
```

Comment:
I think you shouldn't change this test but add a new one for the format with the comma.
Both formats should work.